### PR TITLE
chore: cherry-pick unit test to test against static json

### PR DESCRIPTION
**Pull request by snusifer**
https://tracker.mender.io/browse/QA-471

### CODE REVIEW

**Comment by snusifer** [comment]
Coveralls reports some new lines missed, but those are rather superficial. If everyone is happy, then I'd say we merge despite coveralls complaining

File: cherrypicks_test.go [hunk 1]
```diff
@@ -21,12 +20,21 @@ import (
...
 
-	versionsUrl = ""
-	warnString := fmt.Sprintf(apiWarningString, versionsUrl)
+func TestSuggestCherryPicks(t *testing.T) {
 
 	gitHubOrg := "mendersoftware"
+	server := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
+		rw.Write([]byte(versionsResponse))
+	}))
+	versionsUrl = server.URL
```
Comment (lluiscampos) [inline cherrypicks_test.go:37]: Excellent :guitar:

File: cherrypicks_test.go [hunk 2]
```diff
@@ -21,12 +20,21 @@ import (
 	"github.com/mendersoftware/integration-test-runner/logger"
 )
 
-func TestSuggestCherryPicks(t *testing.T) {
+const versionsResponse = `{
+	"lts": ["3.0", "3.3"],
```
Comment (lluiscampos) [inline cherrypicks_test.go:24]: Suggestion (nit pick): to make the messages closer to the production ones, order these from most recent to older
```suggestion
	"lts": ["3.3", "3.0"],
```

**Review by lluiscampos — APPROVED** [review]
Really nice :+1:

File: cherrypicks_test.go (hunk 2 above)
Comment (snusifer) [reply cherrypicks_test.go:24]: Ah, so my guess was incorrect 🤣

File: cherrypicks_test.go [hunk 3]
```diff
@@ -217,6 +221,7 @@ Hello :smile_cat: This PR contains changelog entries. Please, verify the need of
 			conf.integrationDirectory = tmpdir
 
 			log := logrus.NewEntry(logrus.StandardLogger())
+			log.Infof(" TEST: %s", name)
```
Comment (snusifer) [inline cherrypicks_test.go:224]: @lluiscampos ****, this line slipped through my eye holes. Make a new PR to revert this line? Woopz

Comment (lluiscampos) [reply cherrypicks_test.go:224]: I saw it but didn't react neither. It doesn't bother me so feel free to leave it. As you wish.

**Pull request merged**